client/app/lib/util/isGroupDisabled.coffee

```diff
@@ -0,0 +1,9 @@
+getGroupStatus = require './getGroupStatus'
+
+expiredStatuses = ['expired', 'past_due', 'canceled']
+
+module.exports = isGroupDisabled = (group) ->
+
+  status = getGroupStatus group
```

Comment:
we can do the same for this `status` check again in remote-extension of `JGroup`